app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,28 @@
+# Controller for tag features
+class TagsController < ApplicationController
+  before_action :check_feature_flags
+
+  def index
+    @tags = ActsAsTaggableOn::Tag.all
+  end
+
+  def show
+    @tag_name = params[ :tag ]
+    @tag = ActsAsTaggableOn::Tag.find_by( name: @tag_name )
+    @content_types = tagged_content_types
+    @tagged_items = {}
+    @content_types.each do |type|
+      @tagged_items[ type ] = type.constantize.tagged_with( @tag.name )
+    end
+  end
+
+  private
+
+  def tagged_content_types
+    %w[ BlogPost ]
```

Comment:
Browsing the acts_as_taggable source, it looks like it adds the class method `taggable?` to all models, so if you wanted to avoid having to keep this array up to date, consider something like:

```
def tagged_content_types
  # Select all model classes that return SomeModel.taggable? as true:
  ApplicationRecord.descendants.select(&:taggable?)
end
```